### ISSUE TITLE
Documentation issue with Auto Shift?

### ISSUE THREAD

**Issue by kshmelkov** [qmk/qmk_firmware#1965]
Hello,

Recently I have tried to setup Auto Shift and even though I followed docs it didn't work. After some time I discovered that the correct line for `rules.mk` is

`AUTO_SHIFT_ENABLE = yes`

while the documentation suggests uppercase `YES` which didn't work for me. Surprisingly it didn't complain during compilation about this (should it?) I suppose it is a documentation typo?

**Comment by jcowgar** [comment]
That is correct, sorry about that. I'll make the doc change and submit a pull request.
